Gemfile

```diff
@@ -4,13 +4,12 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
 gem 'i18n', '~> 0.6.11'
 gem 'i18n-js', '~> 3.6.0'
-gem 'rails', '~> 3.2.22'
-gem 'rails-i18n', '~> 3.0.0'
+gem 'rails', '~> 4.0.0'
```

Comment:
Is there a specific reason why we don't point to the latest patch? I found out the latest from Rails 4 series is [4.0.11.1](https://weblog.rubyonrails.org/2014/11/19/Rails-4-0-11-1-and-4-1-7-1-have-been-released/). There are security fixes we should benefit from.

Reply:
I think latest is https://rubygems.org/gems/rails/versions/4.0.13
I'll give it a try now

Reply:
the build is green, will upgrade :+1: